simulation/pathfinder/main.cc

```diff
@@ -1,19 +1,129 @@
 #include "Position.hpp"
 #include "Queue.hpp"
 #include <vector>
+#include <stdio.h>
+#include <deque>
 
-std::vector<Position> pathPos = {Position{12,0}, Position{11,0}, Position{10,0},
-                        Position{9,0}, Position{8,0}, Position{7,0}, Position{7,1},
-                        Position{7,2}, Position{7,3}, Position{7,4}, Position{7,5},
-                        Position{7,6}};
+std::vector<Position> pathPos = {
+   Position{6,2}, Position{6,3}, Position{7,3}, Position{8,3}, Position{9,3}, Position{9,4}, Position{8,4}, Position{7,4}, Position{7,5}, Position{7,6}
+};
 
 Queue<Position> queue;
 
+enum direction{F_U = 'F_U',F_D = 'F_D',cc90 = 'cc90',cw90 = 'cw90', V = 'V', H = 'H'} ;
```

Comment:
I don't know why the compiler doesn't throw error but they has to be a single character each.
Let's make it `enum direction{F_U = 'U',F_D = 'D',cc90 = 'L',cw90 = 'R', V = 'V', H = 'H'} ;` for this moment.

Reply:
Does `V` means it is moving vertically, changing `row` value and vice versa right?

Reply:
V means moving vertically.
the reason why i set the initial value as vertical is that mouse always moves vertically first.
If mouse makes right or left turn the value changes

Reply:
Make sense. Please double check if the ouput is correct as I mentioned in the review summary.